src/components/timing/TimingControls.vue

```diff
@@ -96,23 +112,55 @@ const handleStop = async () => {
       <v-container class="w-100 pa-0 d-flex flex-column align-center">
         <v-sheet class="w-100 d-flex flex-column justify-space-around ma-0" max-width="400">
           <div v-if="!dateStorage.timeIsRunning">
-            <h3>Project</h3>
-            <v-select
-              v-if="projectsStorage.currentCustomerProjects.length >= 1"
-              label="Select Project"
-              :items="projectsStorage.currentCustomerProjects"
-              v-model="projectField"
-              :menu-icon="ArrowDown"
-              :rules="[rules.required]"
-            ></v-select>
-            <p v-else>
-              <v-btn class="ma-0 pa-0" variant="plain" color="black" :to="{ name: 'projects' }"
-                >Add a new project</v-btn
+            <h3 class="text-h5 mb-5">Project</h3>
```

Comment:
This title and "notes" title must have the same style (Now are different). Why use and h3 tag and the change it on the class for an h5?

Reply:
styles corrected :heavy_check_mark: